app/code/Magento/Developer/Console/Command/TemplateHintsShowCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ConfigResource\ConfigInterface;
+
+class TemplateHintsShowCommand extends Command
+{
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:template-hints:show';
```

Comment:
I'm not sure if it's not better to have `:status` instead of `:show`

Reply:
Like for `deploy:mode:show` command, I've preferred `:show` instead of `:status`